### DISCUSSION TITLE
polyphony

### DISCUSSION THREAD

**Discussion by steveparkerdj** [DISTRHO/Cardinal#327] [General]
when i make a poly patch when i do chords i can only hear max of 4 voice notes... 
even if i do do poly for 16notes ... when i test it thru a vca i only hear 1,2,3,4 and after goes silence and end of cycle i get again poly 1,2,3,4
i hope i wrote it clear enough :) 

i tested in vst3 and au ... mac catalina intel 
thanks

**Comment by falkTX** [comment]
This seems like a general question, not a bug.
I am converting this into a discussion.

And no, I didnt fully get it..
Sharing a patch to test the behaviour helps. You can also try to reproduce with the web version at https://cardinal.kx.studio/ to see if it is an issue of plugin format and OS, or just how Cardinal works.

**Comment by falkTX** [comment]
For the example video https://user-images.githubusercontent.com/105745595/180605931-d9b42bc7-fb53-44df-a10d-8b12d42bfc49.mp4

**Comment by falkTX** [comment]
@steveparkerdj what is the polyphony set on the host midi module? sounds like it is set at 4, so that is the limit. did you try changing it to 16?

**Comment by steveparkerdj** [comment]
is set to 16

**Comment by steveparkerdj** [comment]
also.. when i press a chord with my beat step pro input keys i get a normal chord on the 1st input, after i get a different chord even when i press the same chord, after the 3rd strike i get back to the normal chord.

**Comment by dromer** [comment]
Please share a patch, because I don't follow the issue you are describing.

**Reply by steveparkerdj** [reply]
the example video is above ... im not going to install another browser cause my safari doesn't open it , so i can test a patch on browser.
anyway ... ill just not use that module brand cause seems they are all stuck on polyphony notes 
thanks

**Reply by dromer** [reply]
The issue is only with the Aaron Static module right? If I connect straight to Macro Oscillator 2 I can get 16 voices, but with the ScaleCV in between it doesn't go beyond the first 7.

However this seems to be as intended, because the Aaron Static manual says: "Generates a scale and outputs a polyphonic 1v/oct signal with 7 notes".

**Comment by steveparkerdj** [comment]
Ok thanks for the heads up 
Ill not use it so much anyway ... I'm just reporting bugs or errors that I always come across so this great thing can become cleaner :)

**Reply by dromer** [reply]
It's not a bug or error, though.
